client/app/components/composites/ManageAvailabilityHeader/ManageAvailabilityHeader.css

```diff
@@ -0,0 +1,35 @@
+.root {
+  position: relative;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  overflow: hidden;
+}
+
+.colorLayer {
```

Comment:
Usually a pseudo element is a simpler and more expandable solution for these kinds of problems, but here I couldn't use one since the custom color has to be inlined from the JS side.

But the box shadow might work if I'm able to put the blur filter somewhere there. Thanks, I'll try it out.